Sources/SecureStore/KeyManagerService.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import LocalAuthentication
 
 class KeyManagerService {
     let defaultsStore: DefaultsStore
```

Comment:
@Oliver-Binns @benmckillop Does this even need to be part of this class..? Looks like it's only used in `SecureStoreService`